unlocks.csv

```diff
@@ -403,34 +386,42 @@ Analysis|PlasteelIngot|0,0,PlasteelIngot
 Analysis|Beacon|0,0,Beacon
 Analysis|Gravsphere|0,0,Gravsphere
 Analysis|LaserCutter|0,0,LaserCutter
+Goal|OnCraftBattery|0,1,Battery
+Goal|OnCraftComputerChip|0,1,ComputerChip|CopperWire
+Goal|OnCraftTank|0,1,Tank
+Goal|OnCraftDoubleTank|0,1,DoubleTank|Glass|Tank
+Goal|UnlockHatchingEnzymes|0,1,HatchingEnzymes
+Goal|UnlockRadiationSuit|0,1,
+Goal|UnlockRadiationSuit|1,1,
+Goal|UnlockRadiationSuit|2,1,
+Goal|Aurora_RingRoom_Terminal2|0,1,Welder|LaserCutter|Battery
+Goal|Aurora_RingRoom_Terminal3|0,1,Welder|LaserCutter|Battery
+Goal|Precursor_LavaCastleBase_DataDownload1|0,1,
+Goal|Precursor_LavaCastleBase_DataDownload1|1,1,
 Goal|OnPlayRadioGrassy21|0,0,Radio|Welder|Builder|Battery
 Goal|OnPlayRadioGrassy21|1,0,Radio|Welder|Builder|Battery
 Goal|OnPlayRadioGrassy25|0,0,Radio|Welder|Builder|Battery
 Goal|OnPlayRadioMushroom24|0,0,Radio|Welder|Builder|Battery
 Goal|OnPlayRadioSecondOfficer|0,0,Radio|Welder|Builder|Battery
 Goal|OnPlayRadioKoosh26|0,0,Radio|Welder|Builder|Battery
 Goal|OnPlayRadioBloodKelp29|0,0,Radio|Welder|Builder|Battery
-Goal|Aurora_RingRoom_Terminal2|0,1,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Precursor_LavaCastleBase_DataDownload1|0,1,
-Goal|Precursor_LavaCastleBase_DataDownload1|1,1,
-Goal|Aurora_RingRoom_Terminal3|0,1,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
 Goal|OnPlayRadioBounceBack|0,0,Radio|Welder|Builder|Battery
 Goal|Lifepod3|0,0,
 Goal|LifepodSeaglide|0,0,
 Goal|LifepodSeaglide|1,0,
 Goal|LifepodSeaglide|2,0,
 Goal|InnerBiomeWreckLore4|0,0,
 Goal|InnerBiomeWreckLore7|0,0,
-Goal|Aurora_Living_Area_PDA1|0,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Aurora_Locker_PDA2|0,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Aurora_Office_Terminal1|0,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Aurora_Office_Terminal1|1,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Aurora_Cargo_PDA1|0,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Aurora_Cargo_PDA1|1,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Aurora_Living_Area_PDA3|0,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Aurora_Living_Area_PDA3|1,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Aurora_Living_Area_PDA3|2,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
-Goal|Aurora_RingRoom_Code_PDA|0,0,RadiationSuit|RadiationGloves|RadiationHelmet|Welder|PropulsionCannon|LaserCutter|Battery
+Goal|Aurora_Living_Area_PDA1|0,0,Welder|LaserCutter|Battery
```

Comment:
_Technically_ you can access the Aurora without the radiation suit. You just need a safety base nearby, and preferably some free medkits.